### PR TITLE
Add option to require multiple users to approve transfers

### PR DESCRIPTION
**Pull request by michielderoos**
**Describe the Pull Request**

Adds option to require multiple users to approve transfers  

**Todo**

- [x] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [x] Request review from relevant @person or team
- [x] QA your pull request. This includes running the app, login, testing changes, checking [accessibility](https://www.a11yproject.com/checklist/) etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver
- [ ] Add release notes to [unreleased changelog](https://github.com/teamsempo/SempoBlockchain/blob/master/CHANGELOG.md#unreleased)

### CODE REVIEW

**Comment by michielderoos** [comment]
[[High] (12 hrs) Disbursement Approval](https://trello.com/c/fvx8iSwN/788-high-12-hrs-disbursement-approval)

File: app/client/components/creditTransfer/bulkTransferList.tsx
Comment (tristanhcole) [inline app/client/components/creditTransfer/bulkTransferList.tsx:98]: This colour is a bit hard to read - can we change it?

https://ant.design/components/tag/

File: app/client/components/creditTransfer/creditTransferList.jsx
Comment (tristanhcole) [inline app/client/components/creditTransfer/creditTransferList.jsx:505]: See [above](https://github.com/teamsempo/SempoBlockchain/pull/666/files#r614648928).

File: app/client/components/organisation/OrganisationForm.tsx
Comment (tristanhcole) [inline app/client/components/organisation/OrganisationForm.tsx:301]: superadmins or admins? it's not clear to me as a user

File: app/client/components/pages/singleBulkDisbursementPage.jsx
Comment (tristanhcole) [inline app/client/components/pages/singleBulkDisbursementPage.jsx:142]: lol, being explicit makes more sense!

File: app/server/api/credit_transfer_api.py
Comment (tristanhcole) [inline app/server/api/credit_transfer_api.py:137]: Update response message to `Transfer status is {}. Must be PENDING or PARTIAL to modify`

File: app/migrations/versions/f8ce3abf47fb_.py
Comment (tristanhcole) [inline app/migrations/versions/f8ce3abf47fb_.py:1]: don't have an opinion on this- but would it make sense to merge all three migrations into one?

File: app/server/api/credit_transfer_api.py
Comment (tristanhcole) [inline app/server/api/credit_transfer_api.py:315]: If we don't use `automatically_resolve_complete` anymore, should we consider deprecating it?

**Review by tristanhcole — COMMENTED** [review]
A few comments:
- When I create a bulk disbursement and approve it with a sempoadmin, then approve it with an admin - transfers remain as pending, even though the bulk disbursement is approved. Screen Shot 2021-04-16 at 4 46 16 pm

- Need to update partial colour tag on `singleCreditTransfer.jsx` and `BeneficiaryLiveFeed.jsx`.
Screen Shot 2021-04-16 at 4 35 43 pm

- Need to ensure migrations work with current master branch (I had to clear database).

Overall, looks good to me. Tested and works well.

File: app/server/api/disbursement_api.py
Comment (tristanhcole) [inline app/server/api/disbursement_api.py:218]: I think this is the guilty line.

Comment (michielderoos) [reply app/server/api/disbursement_api.py]: That makes sense. The original spec was that anyone could draft a bulk disbursement, but a superadmin had to actually make the transactions. I'm not entirely happy with this solution, but I changed it to `if g.active_organisation.require_multiple_transfer_approvals or AccessControl.has_sufficient_tier(g.user.roles, 'ADMIN', 'superadmin')`, just because when requiring multiple approvers is disabled we probably still want to limit disbursement ability to superadmin.

File: app/server/api/credit_transfer_api.py
Comment (michielderoos) [reply app/server/api/credit_transfer_api.py:315]: I think out of scope, but we should probably do it in the future IMO

File: app/client/components/creditTransfer/bulkTransferList.tsx
Comment (michielderoos) [reply app/client/components/creditTransfer/bulkTransferList.tsx]: Went with the `gold` border colour, looks pretty good! 
image

File: app/client/components/pages/singleBulkDisbursementPage.jsx
Comment (michielderoos) [reply app/client/components/pages/singleBulkDisbursementPage.jsx:156]: It's to keep `PARTIAL` statuses from disabling the button

File: app/client/components/creditTransfer/bulkTransferList.tsx
Comment (tristanhcole) [reply app/client/components/creditTransfer/bulkTransferList.tsx]: Nice!
Nit: do you think we make the gold the same as the `Vendor` on accounts page? -> `#e2a963`

File: app/client/components/pages/singleBulkDisbursementPage.jsx
Comment (tristanhcole) [reply app/client/components/pages/singleBulkDisbursementPage.jsx:156]: yup

File: app/server/api/credit_transfer_api.py
Comment (tristanhcole) [reply app/server/api/credit_transfer_api.py:315]: ok

File: app/client/components/creditTransfer/bulkTransferList.tsx
Comment (tristanhcole) [reply app/client/components/creditTransfer/bulkTransferList.tsx]: Just realized that would make it the same as `PENDING` status. I think leave as is for now then

**Review by tristanhcole — APPROVED** [review]
LGTM!!

File: .circleci/config.yml
Comment (tristanhcole) [inline .circleci/config.yml:240]: remove before merge with master

**Pull request merged**
